### PR TITLE
Series a blog

### PR DESCRIPTION
**Pull request by ryw**
# Notes for blog or docs authors:

-   The `image: ` field in frontmatter of blogs should always be a `png` (`svg` will not work for social previews)
-   You must also duplicate the asset used in `image :` inside of the [public folder](https://github.com/tembo-io/website/tree/main/public)
-   Please write a `description: ` in the frontmatter of any new blog or doc
-   Read more [here](https://github.com/tembo-io/website?tab=readme-ov-file#writing-a-blog-post-%EF%B8%8F)

### CODE REVIEW

File: src/content/blog/2024-06-27-series-a/index.mdx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+* a lot more
+
+This provides teams the ability to build modern data products with maximum security, 
+real time data access, and absolute data sovereignty.
+
+Developers will easily be able to deploy different flavors of Postgres
+to multiple clouds and regions, with advanced options such as auto-scaling,
+auto-pausing, and auto-tuning under active development.
+
+[![cloud.tembo.io](./tembo-ui.png "cloud.tembo.io")](https://cloud.tembo.io)
```
Comment (DarrenBaldwin07) [inline src/content/blog/2024-06-27-series-a/index.mdx:80]: We have a prettier screenshot for this
tembo-ui

**Review by DarrenBaldwin07 — APPROVED** [review]
LFG!

File: src/content/blog/2024-06-27-series-a/index.mdx (hunk 1 above)
Comment (ryw) [reply src/content/blog/2024-06-27-series-a/index.mdx]: @DarrenBaldwin07 what is URL to that image?

Comment (DarrenBaldwin07) [reply src/content/blog/2024-06-27-series-a/index.mdx]: I'll just push it to ur branch real quick since better to use raw svg

**Pull request merged**